GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -428,7 +428,7 @@ public InstanceData parseInstance(Sheet sheet, int startRow,
 
                         case DATE:
                             String dateString = ExportImportUtils.parseCellAsString(cell);
-                            Date date = ExportImportUtils.parseDate(dateString);
+                            Date date = ExportImportUtils.parseSpreadsheetDate(dateString);
                             if (date != null) {
                                 val = String.valueOf(date.getTime());
```

Comment:
I'm not sure that would fix the issue. Note that TZ would still be present, so the `Date` parsing is taking that into account. Resetting the hours/minutes/seconds does not guarantee the output date will be the same as the one introduced in the spreadsheet